KubeArmor/Makefile

```diff
@@ -12,10 +12,12 @@ DLV_LPORT       := 2345
 DLV_RPORT       := $(shell expr $(DLV_LPORT) + $(NETNEXT))
 KUBEARMOR_PID    = $(shell pgrep kubearmor)
 
-ifeq (, $(shell which govvv))
-$(shell go install github.com/ahmetb/govvv)	# This works for older go version
-$(shell go install github.com/ahmetb/govvv@latest) # This works for new go version
-endif
+
+re-govvv:
+	@if ! which govvv > /dev/null; then \
+					echo "installing govvv"; \
+			go install github.com/ahmetb/govvv; \
+	fi
```

Comment:
@Stan370 can you please remove these lines you've added and try? The logic for detecting and installing govvv is already present in lines 15-18 and would install it in every case. I don't see why it might not work unless your Go installation is improper.

Reply:
I initially believed that utilizing govvv during build time was incorrect, only to realize later that the issue stemmed from using a Go version prior to 1.21.